src/commands/setup.rs

```diff
@@ -29,7 +33,66 @@ impl Setup {
             Err(e) => panic!("{}", e),
         };
 
+        // Iterate networks that will require iptables rules
+        for (_, v) in _network_options.network_info {
+            let network_name = v.network_interface.unwrap();
+            // for nets in v.subnets {
+            if let Some(nets) = v.subnets{
+                self.setup_iptables(&network_name, nets);
+            }
+        }
         debug!("{:?}", "Setup complete");
-        //()
+    }
+    /// Set up required iptables for NAT
+    ///
+    ///
+    pub fn setup_iptables(&self, network_name: &str, subnets: Vec<Subnet>) {
+        debug!("iptables");
+        for net in subnets{
+            // Basic object setup
+            let ipt = iptables::new(false).unwrap();
```

Comment:
not-a-blocker: unwrap from serializers might be okay but for iptables we might wanna handle error. Can revisit this later.